test/Microsoft.AspNet.Routing.Tests/Template/TemplateBinderTests.cs

```diff
@@ -14,116 +16,97 @@ public class TemplateBinderTests
     {
         private readonly IInlineConstraintResolver _inlineConstraintResolver = GetInlineConstraintResolver();
 
-        public static IEnumerable<object[]> EmptyAndNullDefaultValues
-        {
-            get
+        public static TheoryData EmptyAndNullDefaultValues =>
+            new TheoryData<string, RouteValueDictionary, RouteValueDictionary, string>
```

Comment:
.. BTW I do agree `IEnumerable<object[]>` -> `TheoryData<string, RouteValueDictionary, RouteValueDictionary, string>` is a real improvement. Just lose the personal preference syntax.